sqlitecluster/SQLite.cpp

```diff
@@ -284,6 +285,20 @@ void SQLite::_sqliteLogCallback(void* pArg, int iErrCode, const char* zMsg) {
         // 17 is the length of "conflict at page" and the following space.
         const char* offset = strstr(zMsg, "conflict at page") + 17;
         _conflictPage = atol(offset);
+
+        // 17 is the length of "part of db table" and the following space.
+        const char* tableOffset = strstr(zMsg, "part of db table") + 17;
```

Comment:
You can start from `offset`, we already know `tablerOffset` will be after that.

Also, renaming `offset` to `pageOffset` might be clearer.

Reply:
Done